HttpClient/SecondApp/src/app/http-service.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams, HttpHeaderResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {Post} from './post';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HttpServiceService {
+  url = 'https://jsonplaceholder.typicode.com';
+
+  constructor(private http: HttpClient) { }
+
+  getAllPosts(): Observable<any>{
```

Comment:
```suggestion
  getAllPosts(): Observable<any[]>{
```
or
```suggestion
  getAllPosts(): Observable<Post[]>{
```

if server returns array